components/automate-cs-ocid/habitat/plan.sh

```diff
@@ -0,0 +1,72 @@
+#shellcheck disable=SC2034
+#shellcheck disable=SC2039
+#shellcheck disable=SC2154
+#stable channel
+
+pkg_name="automate-cs-ocid"
+pkg_description="Wrapper package for chef/ocid"
+pkg_origin="chef"
+# WARNING: Version managed by .expeditor/update_chef_server.sh
+pkg_version="15.4.0"
+vendor_origin="chef"
+pkg_maintainer="Chef Software Inc. <support@chef.io>"
+pkg_license=("Chef-MLSA")
+pkg_upstream_url="https://www.chef.io/automate"
+pkg_deps=(
+  chef/mlsa
+  "${local_platform_tools_origin:-chef}/automate-platform-tools"
```

Comment:
Are these packages really required?

Reply:
We might need to use secret-helper functionality from this package.